front-end/app/src/pages/login/Login.js

```diff
@@ -1,5 +1,10 @@
-export default function LoginPage() {
-  return (
-    <div>login page</div>
-  );
+import React from "react";
+import Form from "./Form";
+import './index.css';
+export default class Login extends React.Component {
+  render() {
+    return(
+      <Form />
```

Comment:
Why do we need a Form object within the login page? Why not just have the code of the form data within the Login Page?